프로그래머스/1/12940. 최대공약수와 최소공배수/최대공약수와 최소공배수.js

```diff
@@ -0,0 +1,5 @@
+function solution(n, m) {
+    const gcd = (num1, num2) => num1 % num2 === 0 ? num2 : gcd(num2, num1 % num2);
+    const lcm = (num1, num2) => num1 * num2 / gcd(num1, num2);
+    return [gcd(n, m), lcm(n, m)];
```

Comment:
```suggestion
function gcd(a, b) {
  while (b != 0) {
    let temp = b;
    b = a % b;
    a = temp;
  }
  return a;
}

function lcm(a, b) {
  return (a * b) / gcd(a, b);
}

function solution(n, m) {
  return [gcd(n, m), lcm(n, m)];
}
```
이런식으로 분리해서 코드를 작성해보는것도 나쁘지 않을것 같아효 ..? ㅎㅎ
하지만 도담님이 작성하신 풀이에서 최소공배수와 최대 공약수 구하는 변수를 적절하게 잘 사용하신 것 같아서 보기에도 편하네요 !